packages/core/contracts/insured-bridge/BridgePool.sol

```diff
@@ -391,7 +384,11 @@ contract BridgePool is Testable, BridgePoolInterface, ExpandedERC20, MultiCaller
         updateAccumulatedLpFees();
         allocateLpFees(_getAmountFromPct(relay.realizedLpFeePct, _depositData.amount));
 
-        emit RelaySettled(depositHash, keccak256(getRelayAncillaryData(_depositData, relay)), msg.sender);
+        emit RelaySettled(depositHash, relayHash, msg.sender);
+
+        delete relay.realizedLpFeePct;
```

Comment:
Why delete if you can't do anything with a `finalized` relay?

Reply:
Gas + being a good ETH citizen and reducing overall state bloat :)

Reply:
This saves gas...?